### DISCUSSION TITLE
Welcome to napalm-logs Discussions!

### DISCUSSION THREAD

**Discussion by mirceaulinic** [napalm-automation/napalm-logs#359] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by chet-space** [comment]
Hey. Happy to see some activity here!
